packages/dds/tree/src/test/shared-tree/schematizingTreeView.spec.ts

```diff
@@ -164,19 +180,39 @@ describe("SchematizingSimpleTreeView", () => {
 
 		assert.deepEqual(log, [["rootChanged", 5]]);
 
-		assert.equal(view.error, undefined);
+		assert.equal(view.compatibility.isEquivalent, true);
 		assert.equal(view.root, 5);
 	});
 
-	it("Open incompatible document", () => {
+	it("Attempt to open document using view schema that allows a subset of stored schema documents", () => {
```

Comment:
I don't think this phrasing is accurate. View schema defined an editing API, so a view schema is only compatible with a stored schema if the set of documents both are compatible with are the same.

Saying a view schema is incompatible with a stored schema makes sense (the sets are not the same), but you can't really say a view schema is compatible with a subset of stored schema documents, since no possible document using that stored schema can be opened using the view schema since for any of them, an edit that goes out of schema would be permitted.

If we supported readonly views, and we allowed the view to deeply inspect int document for compatibility based on the content and not its stored schema (we support neither of those) then I think this statement would be well defined.

Before your change "incompatible" was used to mean, cannot be viewed or upgraded to match view schema. I think thats how you are still using incompatible below, it just seems like you are trying two different incompatible cases?

```suggestion
	it("Attempt to open document using view schema that incompatible due to being too strict compared to stored the schema", () => {
```